util/titles.py

```diff
@@ -0,0 +1,62 @@
+from nameparser import HumanName
+from nose.tools import set_trace
+import re
+
+from permanent_work_id import WorkIDCalculator;
+
+
+"""Fallback algorithms for dealing with personal names when VIAF fails us."""
+
+def is_corporate_title(display_name):
```

Comment:
What do you use `is_corporate_title` for? I don't see it mentioned elsewhere in the code. It doesn't seem to me like we want to do something special if the title of a book looks like the name of a company.